views/pages/error.pug

```diff
@@ -1,6 +1,35 @@
-doctype html
-html(lang="en")
-    include ../partials/header.pug 
-    body 
-        p Hello World! 
-        p The webpage you are looking for does not exist!
\ No newline at end of file
+extends /views/templates/frontend-base.pug
+include /views/mixins/pink-button.pug
+
+append styles
+    link(href="/assets/css/pages/error.css" rel="stylesheet")  
+    link(rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css")
+
+block body 
+    h1 Oops, This Page Could Not Be Found!
+    section(class="error-main")
+        p(class="error-404") 404
+        div(class="error-links")
+            h2 Helpful Links 
+            ul 
+                li 
```

Comment:
kind of a nitpick, but since this is basically a navbar, could we use the navbar data here instead of rebuilding a menu? all pages are passed the nav data from the server, so you could replace this ul with something like:
```
ul
   each item in navItems
        li
             i.fa.fa-angle-right
             a(href=item.ref) #{item.title}
```
this way, updates to the `navbar.json` file would also be reflected here without us having to manually update the links/titles of this menu. note that you can access navbar data with `navItems`, take a look at `views/partials/navbar.pug` for a reference.